_data/projects/Cesium.yml

```
@@ -0,0 +1,11 @@
+name: Cesium
+desc: C compiler for the CLI platform
+site: https://github.com/fornever/cesium
+tags:
+- .net
+- compiler
+- c
+- c#
+upforgrabs:
+  name: up for grabs
```

:art: label must match the value used

```suggestion
  name: up-for-grabs
```